product-scenarios/3-securing-an-api-with-different-security-protocols/3.2-api-developers-want-to-secure-their-api-using-an-user-access-control-machanism/3.2.1-api-developer-wants-to-secure-api-resources-using-user-roles/pom.xml

```
@@ -43,7 +39,7 @@
                 <configuration>
                     <argLine>-Xmx1024m -XX:PermSize=256m -XX:MaxPermSize=512m</argLine>
                     <suiteXmlFiles>
-                        <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
+                        <suiteXmlFile>${suiteXmlFile}</suiteXmlFile>
```

Same as above. Ignore this for now
```suggestion
                        <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
```